Food_Tracker_App/app/src/main/java/com/example/foodtracker/Ingredient.java

```diff
@@ -0,0 +1,67 @@
+package com.example.foodtracker;
+
+public class Ingredient {
+    private String description;
+    private int quantity;
+    private String location;
+    private String category;
+    private int amount;
```

Comment:
Could this be in `BigDecimal` or something that can hold decimal numbers? `int` data type only allows whole numbers